newspaper/extractors.py

```diff
@@ -216,6 +216,12 @@ def parse_date_str(date_str):
              'content': 'content'},
             {'attribute': 'pubdate', 'value': 'pubdate',
              'content': 'datetime'},
+            {'attribute': 'name', 'value': 'publish-date',
+             'content': 'content'},
+            {'attribute': 'name', 'value': 'created-date',
+             'content': 'content'},
+            {'attribute': 'name', 'value': 'modified-date',
```

Comment:
I'd be careful with modified-date - I've recently been stung where the true publication date was early 2000s whereas the modified date was a recent date due to updating copyright or some other unrelated content